### PR TITLE
Add TCB 58 about Mitzu

### PR DESCRIPTION
**Pull request by mosabua**
Preview at https://deploy-preview-638--trino.netlify.app/episodes/58

Timestamps, yt, wistia and more to be done after live stream as usual.

also fyi @IstvanM

### CODE REVIEW

File: _episodes/58.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* [Manfred Moser](https://www.linkedin.com/in/manfredmoser), Director of Trino
+  Community Leadership at [Starburst](https://starburst.io),
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* [István Mészáros](https://www.linkedin.com/in/imeszaros/), CEO and founder of
```
Comment (mosabua) [inline _episodes/58.md:30]: Please confirm links here @IstvanM

File: _episodes/58.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  shipped with a new user interface thanks to a contribution from our new
+  [Starburst Trino champion Peng
+  Wei](https://www.starburst.io/community/trino-champions/)
+* Status of the continuous integration and build setup with Apache Maven
+  improved a lot thanks to our collaboration with the new [Starburst Trino
+  champion Tamas Cservenak](https://www.starburst.io/community/trino-champions/)
+* Trino Contributor Call [recap is now
+  available](https://github.com/trinodb/trino/wiki/Contributor-meetings#trino-contributor-call-21-mar-2024)
+
+## Mitzu
```
Comment (mosabua) [inline _episodes/58.md:66]: We can change title and content for this whole section as you like @IstvanM .. just comment or let me know on slack

File: _episodes/58.md (hunk 1 above)
Comment (IstvanM) [reply _episodes/58.md:30]: yes, this is correct.

File: _episodes/58.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+In our conversation and demo session with István we cover all the following
+aspects and a few other topics:
+
+* What is product analytics?
+* Discuss some key terms such as segmentation, funnels, retention, and revenue.
+* What are some example use cases?
+* What kind of products can be analyzed?
+* Use of Mitzu for marketing.
+* What insights can you gain?
+* What other product analytics tools exist, and why is Mitzu better?
```
Comment (colebow) [inline _episodes/58.md:89]: Maybe frame this as "what sets Mitzu apart" instead of "why is Mitzu better" just so we're taking less strong of a stance?

**Review by colebow — COMMENTED** [review]
Honestly all looks good. Will approve once timestamps are in.

File: _episodes/58.md (hunk 3 above)
Comment (IstvanM) [reply _episodes/58.md:89]: agree

Comment (IstvanM) [reply _episodes/58.md:89]: We know that in many cases traditional product analytics is better for companies.

File: _episodes/58.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+<a href="{{site.url}}/ecosystem/client.html#mitzu">
+  <img src="{{site.url}}/assets/images/logos/mitzu.png">
+</a>
+
+In our conversation and demo session with István we cover all the following
+aspects and a few other topics:
+
+* What is product analytics?
+* Discuss some key terms such as segmentation, funnels, retention, and revenue.
```
Comment (IstvanM) [inline _episodes/58.md:84]: I would remove revenue, we won't have time for it.

File: _episodes/58.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+* What are some example use cases?
+* What kind of products can be analyzed?
+* Use of Mitzu for marketing.
+* What insights can you gain?
```
Comment (IstvanM) [inline _episodes/58.md:88]: This is going to be very similar to what I will most likely tell at the Segmentation, Funnels, Retention part

File: _episodes/58.md (hunk 3 above)
Comment (mosabua) [reply _episodes/58.md:89]: Done

File: _episodes/58.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+  desc: "Mitzu and warehouse-native product analytics"
+  time: 10:36
+- title: "What products and what source data?"
+  time: 12:04
+- title: "More about events, users, and sessions"
+  time: 13:30
+- title: "Mitzu use cases, personas, and difference to BI tools"
+  time: 15:30
+- title: "Common and supported event data structures"
+  time:  17:10
```
Comment (colebow) [inline _episodes/58.md:37]: Not sure if it matters but this has an extra bit of spacing.

**Review by colebow — APPROVED** [review]

File: _episodes/58.md (hunk 6 above)
Comment (mosabua) [reply _episodes/58.md:37]: Fixed

**Pull request merged**